src/rules/selector-pseudo-element-no-unknown/README.md

```diff
@@ -0,0 +1,93 @@
+# selector-pseudo-element-no-unknown
+
+Disallow unknown pseudo-element selectors.
+
+```css
+    a::before {}
+/**    ↑
+ * This pseudo-element selector */
+```
+
+All vendor-prefixes pseudo-element selectors are ignored.
+List of valid pseudo-element selectors:
```

Comment:
Oops, I missed his comment first time around. 

I still think it should go as a user just needs know the rule checks for known (i.e. from the specs) rather than exactly what it looks for. We're now also trying to consolidate all the lists *into one place*.

@davidtheclark What do you think?